### PR TITLE
labeler: add ble mesh label

### PR DESCRIPTION
**Pull request by alxelax**
The ble mesh team created PR bot that gathers PRs from different repos to report them in MS Team channel to follow up review state. To operate well PR Bot needs label. This PR adds assigning`ble mesh` label.

Signed-off-by: Aleksandr Khromykh <aleksandr.khromykh@nordicsemi.no>

### CODE REVIEW

**Comment by Andrewpini** [comment]
is it possible to include mesh docs path to this label as well?

**Comment by alxelax** [comment]
> is it possible to include mesh docs path to this label as well?

Sure, could you point path? I didn't find it.

**Comment by Andrewpini** [comment]
> > is it possible to include mesh docs path to this label as well?
> 
> Sure, could you point path? I didn't find it.

I guess:
nrf\doc\_build\nrf\src\libraries\bluetooth_services\mesh
nrf\doc\_build\nrf\src\samples\bluetooth\mesh

there might be some othere places as well, but this covers model implementation and samples

**Review by omkar3141 — APPROVED** [review]

File: .github/labeler.yml [hunk 1]
```diff
@@ -16,3 +16,9 @@
 
 "changelog-entry-required":
   - all: ["!doc/nrf/releases/release-notes-changelog.rst"]
+
+"ble mesh":
+  - "subsys/bluetooth/mesh/*"
+  - "include/bluetooth/mesh/*"
+  - "samples/bluetooth/mesh/*"
+  - "doc/nrf/libraries/bluetooth_services/mesh/*"
```
Comment (PavelVPV) [inline .github/labeler.yml:24]: Mesh user guides are located at `doc/nrf/` and file names start with `ug_bt_mesh_`.

**Review by PavelVPV — APPROVED** [review]

**Comment by alxelax** [comment]
@nrfconnect/ncs-ci,  please scream if these changes are out of some CI policies. Otherwise. I'll ask to merge them.

**Review by thst-nordic — APPROVED** [review]

**Pull request merged**